lib/Analysis/SeaBuiltinsInfo.cc

```diff
@@ -134,6 +137,26 @@ Function *SeaBuiltinsInfo::mkIsDereferenceable(Module &M) {
   return FN;
 }
 
+Function *SeaBuiltinsInfo::mkAssertIf(llvm::Module &M) {
+  auto &C = M.getContext();
+  auto FC = M.getOrInsertFunction(SEA_ASSERT_IF, Type::getInt1Ty(C), Type::getInt1Ty(C), Type::getInt1Ty(C));
+  auto *FN = dyn_cast<Function>(FC.getCallee());
+  if (FN) {
+    FN->setOnlyAccessesInaccessibleMemory();
+    FN->setDoesNotThrow();
+    FN->setDoesNotFreeMemory();
+    FN->setDoesNotRecurse();
+    // TODO: why does nocapture work in sea_is_def and not here?
```

Comment:
this function takes no pointers so it has no ability to capture anything. Hence it cannot have `nocapture` attribute.

Reply:
ack.